tests/test_ds.py

```diff
@@ -0,0 +1,30 @@
+from pymoab import core, types
+from pymoab.rng import Range
+import dagmc_stats.DagmcStats as ds
+
+test_env = [{'input_file': '3vols.h5m'}, {
+    'input_file': 'single-cube.h5m'}, {'input_file': 'pyramid.h5m'}]
+
+
+def test_load_file():
+    """Tests loading file and check the values of all the variables that are set in the constructor
+    """
+
+    single_cube = ds.DagmcStats(test_env[1]['input_file'])
+
+    print(single_cube.root_set)
+    assert(single_cube.root_set == 0)
+
+    print(single_cube._vert_data)
+    print(single_cube._tri_data)
+    print(single_cube._surf_data)
+    print(single_cube._vol_data)
```

Comment:
Rather than printing this information to the screen, we should be checking that this data is what we expect it to be and do something like `assert(single_cube._vert_data == [what we expect it to be])`

Reply:
I would like to discuss this during the meeting. Thanks!